cocos/3d/CCPlane.h

```diff
@@ -100,6 +100,11 @@ class CC_DLL Plane
     */
     PointSide getSide(const Vec3& point) const;
 
+    /**
+    * Return the side where the point is.
+    */
+    PointSide getSide(const Vec3& point, const Vec3& halfSize) const;
```

Comment:
halfSize is confusing. tolerance is better.
And update the comment. Distance between point and plane is greater than tolerance returns PointSide::FRONT_PLANE, distance is less than -tolerance returns return PointSide::BEHIND_PLANE, returns PointSide::IN_PLANE otherwise.